ad/internal/config/config.go

```diff
@@ -247,12 +255,104 @@ func (c *KerberosTransporter) Post(_ *winrm.Client, request *soap.SoapMessage) (
 	defer resp.Body.Close()
 
 	if resp.StatusCode != 200 {
-		return "", fmt.Errorf("http error while making kerberos authenticated winRM request: %s", err)
+		return "", fmt.Errorf("http error while making kerberos authenticated winRM request: %d - %s ", resp.StatusCode, resp.Status)
 	}
 
 	body, err := ioutil.ReadAll(resp.Body)
 	if err != nil {
 		return "", err
 	}
+
 	return string(body), err
 }
+
+// ProviderConf holds structures that are useful to the provider at runtime.
+type ProviderConf struct {
```

Comment:
Should there be a max amount of connections for the pool? Or not really a concern?

Reply:
Not a concern so far.